src/mudblazor/MudBlazor.Template/MudBlazor.Template.csproj

```diff
@@ -31,7 +31,8 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="8.*" Condition="'$(IndividualLocalAuth)' == 'True' AND '$(UseLocalDB)' != 'True'" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.*" Condition="'$(IndividualLocalAuth)' == 'True' AND '$(UseLocalDB)' == 'True'" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.*" Condition="'$(IndividualLocalAuth)' == 'True'" />
-    <PackageReference Include="MudBlazor" Version="7.0.0-preview.4" Condition="'$(UseWebAssembly)' != 'True'"/>
+    <PackageReference Include="MudBlazor" Version="7.0.0-rc.2" Condition="'$(UseWebAssembly)' != 'True'" />
+    <PackageReference Include="Extensions.MudBlazor.StaticInput" Version="2.0.0-rc.2" />
```

Comment:
Can we add an condition that it's added only when `IndividualLocalAuth` is selected?

Reply:
Fixed